src/Components/Notifications/NotificationItem.tsx

```diff
@@ -64,21 +66,30 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ item }) => {
     })
   }
 
+  // Temporary until new activity panel is ready
+  const message = () => {
+    if (item.notificationType === "PARTNER_OFFER_CREATED") {
+      return "Offer"
```

Comment:
**question**: Is this override necessary if the backend `message` is identical ([source](https://github.com/artsy/gravity/blob/4377ac31d091ff3d0b359b2fb81c517f14a66afe/app/models/activity_feed/partner_offer_created_activity.rb#L17))?

Reply:
Yeah, unfortunately we need it here. Metaphysics call `toLowerCase()` on messages ([code](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/notifications/index.ts#L51)). I think we should remove it from there and change gravity to create activities with proper messages. I was just about to create a ticket for this.

Reply:
Gotcha. Thanks for the explanation. 👍🏼 

I suppose we could omit this message entirely depending on what we hear back from Barney regarding designs that fit into the existing structure. As is, it might feel a bit repetitive without much added value.

> Limited Time Offer · Today
> Saved work by Heather Boose Weiss
> ⏱️ Expires in 2 days